test/unlimited_contract_size.js

```diff
@@ -0,0 +1,88 @@
+var Web3 = require('web3');
+var Ganache = require("../index.js");
+var fs = require("fs");
+var path = require("path");
+var solc = require("solc");
+
+// Thanks solc. At least this works!
+// This removes solc's overzealous uncaughtException event handler.
+process.removeAllListeners("uncaughtException");
```

Comment:
These tests likely belong in `vm.js`

Reply:
I disagree. I don't think we should bombard a bunch of "junk" into one file. I would much rather support a folder-based structure.

For example we have `vm.js` and `unlimited_contract_size.js`. I think this would be better as `vm/revert.js` and `vm/unlimited_contract_size.js` rather than putting it all in `vm.js` and causing chaos for maintenance.

Reply:
From @benjamincburns on Slack: